ethcore/src/spec/spec.rs

```diff
@@ -124,7 +126,7 @@ impl CommonParams {
 		schedule.have_static_call = block_number >= self.eip214_transition;
 		schedule.have_return_data = block_number >= self.eip211_transition;
 		if block_number >= self.eip210_transition {
-			schedule.blockhash_gas = 350;
+			schedule.blockhash_gas = 800;
```

Comment:
this value is also set here https://github.com/paritytech/parity/pull/5855/files#diff-368d67dfd535628837d23122934dc5c3R200 . imo, it should be a const

Reply:
The latter function is used only in some tests and the value should not really be set there. I'll remove it